### PR TITLE
Fix HTML to JSX converter to work with https

### LINKED ISSUE #3455: HTMLtoJSX is not defined on HTML to JSX converter page
screen shot 2015-03-18 at 4 12 31 pm

Edits were made last week to this page, maybe it was broken accidentally?
https://github.com/facebook/react/blob/gh-pages/html-jsx.html

### PR DESCRIPTION
**Pull request by cody**
Fixes #3455

### CODE REVIEW

File: docs/html-jsx.md [hunk 1]
```diff
@@ -6,6 +6,6 @@ id: html-jsx
 <div class="jsxCompiler">
   <h1>HTML to JSX Compiler</h1>
   <div id="jsxCompiler"></div>
-  <script src="http://reactjs.github.io/react-magic/htmltojsx.min.js"></script>
+  <script src="https://reactjs.github.io/react-magic/htmltojsx.min.js"></script>
```
Comment (tgolen) [inline docs/html-jsx.md]: Why not
```
<script src="//reactjs.github.io/react-magic/htmltojsx.min.js"></script>
```
so it is protocol agnostic?

**Comment by pirate** [comment]
+1 if you make it // instead of https://

File: docs/html-jsx.md (hunk 1 above)
Comment (pftg) [inline docs/html-jsx.md]: :+1:

Comment (cody) [inline docs/html-jsx.md]: In another PR `//` was used at first, but it was argued that the prefix should be `https//`: https://github.com/facebook/react/pull/3180#issuecomment-74913038

@zpao I am happy to change this, just let me know.

**Comment by spicyj** [comment]
This is good as-is – thanks!

**Comment by Daniel15** [comment]
Thanks - I'm curious as to why this only broke recently though? Is the React site loading via HTTPS now but it never used to?

**Comment by pirate** [comment]
The link to the conversion page is relative, so it'll only break if they originally landed on the github pages domain using https (which is not the default).  I have the https-everywhere extension which is why it broke for me.

**Comment by jimfb** [comment]
Looks like we never merged this to stable.

**Comment by zpao** [comment]
This was landed before the stable branch was cut and you can see this code in the branch.

**Pull request merged**